comparisontool/tests/settings.py

```diff
@@ -0,0 +1,46 @@
+import dj_database_url
+import os
+
+DEBUG = True
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'dccc.sqlite3',
```

Comment:
How would you feel about changing [this line](https://github.com/cfpb/django-college-costs-comparison/blob/master/.gitignore#L41) in the `.gitignore` to ignore all sqlite3 files, similarly to what we do [here](https://github.com/cfpb/open-source-project-template/blob/master/.gitignore#L28)? Otherwise, someone using this code to create a local database (with `./manage.py migrate`) will see their Sqlite database as an uncommitted change.